Makefile

```diff
@@ -1,5 +1,5 @@
 preview:
-	ENV=dev php -S localhost:8000 -t web
+	ENV=dev ./console serve
```

Comment:
`make preview` is the command used in development environment to run a small webserver.

We were previously using PHP's built-in webserver, now we simply launch the React application.